media-libs/sdl2_mixer/sdl2_mixer-2.0.1.recipe

```diff
@@ -23,9 +23,10 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
 	lib:libogg$secondaryArchSuffix
-	lib:libFLAC$secondaryArchSuffix
+	lib:libflac$secondaryArchSuffix
```

Comment:
IIRC there was an issue to use lowercase when possible, mixing them up wouldn't be a good idea, the provides is another thing as using the requirements .. or if I'm wrong please correct me (but I've been corrected in the past for this also -- using uppercase, having a clear understanding on what is prefered would be nice :)